twitter_test.go

```diff
@@ -197,7 +199,7 @@ func Test_GetTweet(t *testing.T) {
 	}
 
 	if tweet.Text != tweetText {
-		t.Fatalf("Tweet %d contained incorrect text. Received: %s", tweetId, tweetText)
+		t.Fatalf("Tweet %d contained incorrect text. Received: %s", tweetId, tweet.Text)
```

Comment:
hah - good catch!

Reply:
This one gave me headaches trying to debug my changes. Until I finally noticed this mistake was misleading me the entire time ;-)